src/ppx/graphics_util.cpp

```diff
@@ -31,6 +35,175 @@
 namespace ppx {
 namespace grfx_util {
 
+namespace {
+
+// Start planar image helper functions
+
+// Gets the height of a single plane, in terms of number of pixels represented.
+// This doesn't directly correlate to the number of bits / bytes for the plane's
+// height. The value returned can be used in a copy-image-to-buffer command.
+// plane: The plane to get the height for (containing information about the
+//        color components represented in the plane).
+// subsampling: The type of subsampling applied to chroma values for the image
+//              (e.g. 444, 422, 420).
+// imageHeight: The height of the image, in pixels, with no subsampling applied.
+uint32_t GetPlaneHeightInPixels(
+    const grfx::FormatPlaneDesc::Plane& plane,
+    grfx::FormatChromaSubsampling       subsampling,
+    uint32_t                            imageHeight)
+{
+    bool hasColSubsampling = (subsampling == grfx::FORMAT_CHROMA_SUBSAMPLING_420);
```

Comment:
Shouldn't the condition be `subsampling != grfs::FORMAT_CHROMA_SUBSAMPLING_UNDEFINED`?

Reply:
Ah got it.